packages/headless/src/features/configuration/configuration-actions.ts

```diff
@@ -58,12 +60,14 @@ export const updateAnalyticsConfiguration = createAction(
     originLevel2?: string;
     originLevel3?: string;
     apiBaseUrl?: string;
+    runtimeEnvironment?: IRuntimeEnvironment;
   }) =>
     validatePayload(payload, {
       enabled: new BooleanValue({default: true}),
       originLevel2: originSchemaOnConfigUpdate(),
       originLevel3: originSchemaOnConfigUpdate(),
       apiBaseUrl: new StringValue({url: true, emptyAllowed: false}),
+      runtimeEnvironment: new Value(),
```

Comment:
Can we completely skip validation instead of doing this ? 

Or is bueno throwing an error if we do not validate every key ?

Reply:
Since we're using the object returned from Bueno directly we need to have all the attributes define for it to be of the correct type. We could use Bueno for validation only and return the original object but I think this is a pretty common pattern in headless and we don't need to change it for just this.